src/sentry/demo/data_population.py

```diff
@@ -1130,10 +1171,37 @@ def handle_react_python_scenario(self, react_project: Project, python_project: P
             )
         self.assign_issues()
 
+    def handle_mobile_scenario(self, ios_project: Project, android_project: Project):
```

Comment:
@roggenkemper I think we are going to have issues with releases because we aren't adding the releases to the mobile projects (see `generate_releases`). It might be worth pulling `generate_releases` out of `handle_react_python_scenario` and call it directly in `create_demo_org`. Either that, or make a new function that will create releases and populate events that we can call in `create_demo_org`.

Reply:
> ![Screen Shot 2021-06-16 at 10 52 57 AM](https://user-images.githubusercontent.com/8533851/122268541-08e90200-ce91-11eb-8e6a-df17d3ba931b.png)
> 
> It looks like the start time is way in the past, we should either remove that field or make it closer to the current time

good point, I removed it.

Reply:
moved call to create_demo_org